BaseFilter/sections/css_extended.txt

```diff
@@ -1946,6 +1946,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-pa
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
 linuxadictos.com#?##content > aside[id^="text-"]:has(> div.textwidget > p.gt-block > ins.adsbygoogle)
+techpowerup.com#?#.v-port > div:has(>div > img[src*="/reviyuu"])
```

Comment:
Better `#?#.v-port > div:has(> a[rel="nofollow"][href^="/reviyuu/"])`. Make the depth as few as possible, i.e. (> * > *) => (> *). Also using src for :has is bit risky as the src may be removed from source code (https://github.com/AdguardTeam/AdguardFilters/issues/95975#issuecomment-961235146)

Reply:
I see. I will fix rule.